winpr/libwinpr/crypto/hash.c

```diff
@@ -365,10 +369,32 @@ BOOL winpr_Digest_Init(WINPR_DIGEST_CTX* ctx, WINPR_MD_TYPE md)
 
 	if (mbedtls_md_starts(mdctx) != 0)
 		return FALSE;
-#endif
 
 	return TRUE;
 }
+#endif
+
+BOOL winpr_Digest_Init_MD5_Allow_FIPS(WINPR_DIGEST_CTX* ctx)
```

Comment:
@akallabeth Sounds like this would be close to what @ondrejholy suggested which I did not like in this case for the reason stated in a previous comment. I was thinking more of a generic "winpr_Digest_Init_Allow_FIPS" vs. a more generic function which could be extended to more flags in the future. I really wanted to force overrides of FIPS through a very obvious and specific API.